src/tests/token/erc721/test_dual721.cairo

```diff
@@ -80,20 +79,23 @@ fn test_dual_name() {
 }
 
 #[test]
+#[ignore]
 #[should_panic(expected: ('ENTRYPOINT_NOT_FOUND',))]
 fn test_dual_no_name() {
     let dispatcher = setup_non_erc721();
     dispatcher.name();
 }
 
 #[test]
+#[ignore]
 #[should_panic(expected: ("Some error", 'ENTRYPOINT_FAILED',))]
```

Comment:
The issue with this kind of expected error can be solved by removing 'ENTRYPOINT_FAILED' error
```
#[should_panic(expected: ("Some error",))]
```
We should consider if we want to go this way instead of ignoring all such tests

Reply:
Good catch.